src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature

```diff
@@ -50,3 +50,31 @@ Feature: Clinical Visit Management
     When a user attaches patient supporting file
     And a user clicks the upload file button
     Then the system uploads the file
+
+    
+  @selenium
+  @clinicalVisit
+  Scenario: Patient Dashboard adding Allergy2
+    # User story: Adding and deleting New Allergy2
+    When a user clicks on Allergy2 link from Patient dashboard page2
+    Then the system loads Allergy2 board page
+    When a user clicks Add New Allergy2 button
+    And the user clicks on the Save button2
+    Then the system adds a new Allergy2
+    When the user clicks on the delete button from the patient dashboard
+    And the system loads Remove Allergy dashboard
+    Then system displays no allergy in the Allergies table
+
+  @selenium
+  @clinicalVisit
+  Scenario: Patient Dashboard adding Condition2
+    # User story: Adding and deleting New Condition2
+    When a user clicks on Condition2 link from Patient dashboard page
+    Then the system loads Condition2 board page
+    When a user clicks Add New Condition2 button
+    And the system loads Add New Condition2 dashboard
+    And user clicks on the Save button2
+    Then the system adds a new Condition2
+    When the user clicks on the delete button from dashboard
+    Then user clicks on the yes button to confirm
+
```

Comment:
@Ndacyayisenga-droid you need only one empty line at the end of the file to cater for git collaborators !

Reply:
@Ndacyayisenga-droid did you see the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/137/files#r722445149)?